src/main/java/net/datafaker/providers/base/DateAndTime.java

```diff
@@ -264,6 +273,25 @@ public Timestamp birthday(int minAge, int maxAge) {
             LocalDateTime.of(date, LocalTime.ofNanoOfDay(faker.number().numberBetween(0, DAYS_NANOS))));
     }
 
+    /**
+     * Generates a random birthday between two ages from now as LocalDate.
+     *
+     * @param minAge the minimal age
+     * @param maxAge the maximal age
+     * @return a random birthday between {@code minAge} and {@code maxAge} years ago from now.
+     * Negative {@code minAge} and {@code maxAge} are supported.
+     */
+    public LocalDate birthdayLocalDate(int minAge, int maxAge) {
+        final LocalDate localDate = LocalDate.now();
+        final LocalDate from = localDate.minusYears(maxAge);
+        if (minAge == maxAge) {
+            return from;
+        }
```

Comment:
I'd still suggest moving this before the var declarations, no point creating them if they won't be used.

Reply:
OMG disregard, you're returning one of the declared values 🤦‍♂️

Reply:
I didn't get it
```java
final LocalDate localDate = LocalDate.now();
final LocalDate from = localDate.minusYears(maxAge);
if (minAge == maxAge) {
    return from;
}
```
the return value is based on the 2 values declared above since first need to know current date and need to substract the input age

Reply:
It’s fine. I was being brain dead for a sec. Sorry.

Reply:
ok, no problem , thanks for taking a look